wallet-sdk/src/main/kotlin/org/stellar/walletsdk/Data.kt

```diff
@@ -165,8 +177,40 @@ data class SignerWeight(
   val recoveryServer: Int,
 )
 
+enum class StellarOperationTypes(val type: String) {
```

Comment:
I don't think this enum is needed you can just typecheck instead (see comment below)